mungegithub/mungers/matchers/comment/finder.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package comment
+
+import "github.com/google/go-github/github"
+
+// Filter let's you retrieve your filtered comments
+type Filter struct {
+	comments []*github.IssueComment
+	matcher  Matcher
+}
+
+// Iterator over the list of filtered comments
+type Iterator struct {
+	direction int
```

Comment:
Does this specify more than just the direction?  It seems to also specify the step count right?  If it's just forward/reverse, does it need to be an int (vs a bool)